docs/getting-started/artifact-control.asciidoc

```diff
@@ -6,8 +6,18 @@
 :frontmatter-tags-content-type: [how-to]
 :frontmatter-tags-user-goals: [secure, manage]
 
-[sidebar]
---
-[.text-center]
-This page is a placeholder for future documentation.
---
+On the **Protection updates** tab of the {elastic-defend} integration policy, you can configure how {elastic-defend} receives updates from Elastic with the latest threat detections, global exceptions, malware models, rule packages, and other protection artifacts. By default, these artifacts are updated automatically every day, ensuring your environment is up to date with the latest protections. 
+
+You can disable automatic updates and freeze your protection artifacts to a specific date, allowing you to control when to receive and install the updates. For example, you might want to temporarily disable updates to ensure resource availability during a high-volume period, or to test updates in a controlled staging environment before rolling out to production.
+
+You cannot disable updates indefinitely — after 18 months, protection artifacts will expire. At that time, {elastic-defend} will start updating with the oldest artifacts available each day, until you either select a more recent version or re-enable automatic updates.
```

Comment:
per our chat on slack - 
They actually can keep the artifact version after 18 months - but it won't be available for a "rollback". So, if they were to change it from the 18 month old date, to the latest (for example), they wouldn't be able to switch it back if something went wrong. So essentially - the calendar picker supports 18 months of artifacts/protections days to select from, if that makes sense.

Reply:
Revised in https://github.com/elastic/security-docs/pull/4261/commits/ce36c7c1aac25541e799512c02ab5cbb86904f2b